d-rita/fast-food-fast-react
# #164353013 Set up react routing

**What does this PR do?**
This PR sets up routing in the application to navigate through components.

**Description of tasks completed**
- Install react-router-dom
- Add routes to the application

**How this can be manually tested**
- Fetch and check out this branch using `git fetch origin ch-set-up-routing-164353013 && git checkout ch-set-up-routing-164353013`
- To install dependencies, run `yarn install`
- In the terminal, use `yarn start` to run the application on localhost.
- Go to` localhost:8080/home` to see a rendered view.

**Related Pivotal Tracker Story**
[#164353013](https://www.pivotaltracker.com/story/show/164353013)
